testpilot-metrics.js

```diff
@@ -24,15 +24,15 @@
  */
 function Metrics({id, version, uid, tid = null, type = 'webextension', debug = false}) {
   if (!id) {
-    throw new SyntaxError(`'id' property is required.`);
+    throw new SyntaxError("'id' property is required.");
```

Comment:
Ha! Turns out that ESLint does have an `allowTemplateLiterals` option for the `quotes` rule.
Fixed.